### ISSUE TITLE
[Accessibility] nav pills tabs : The focus on active link is not visible

### ISSUE THREAD

**Issue by cfagno** [twbs/bootstrap#31427]
On tabs with pills the focus on an **active link is not visible** (see https://v5.getbootstrap.com/docs/5.0/components/navs/) : 
![TabsActiveLinkFocused](https://user-images.githubusercontent.com/67265152/89298125-2d673180-d665-11ea-82b8-8649a050db09.gif)

But on **link button** the **focus is visible** (see https://v5.getbootstrap.com/docs/5.0/components/buttons/) : 

![activeLinkFocused](https://user-images.githubusercontent.com/67265152/89298215-47087900-d665-11ea-9d6c-e5589de2286a.gif)

There is no focus style visible on **Firefox** browser for this active link, so we don't see the focus (we lose where we are).

To be accessible the **focus on an active link must be visible** (see success criterion 2.4.7 : https://www.w3.org/TR/WCAG21/#focus-visible).

**Comment by mayralgr** [comment]
@XhmikosR is this issue still open? can you give me more information? and this is to be corrected in the v5?
Thank you!

**Comment by XhmikosR** [comment]
![image](https://user-images.githubusercontent.com/349621/101259684-f8620d00-3732-11eb-9927-4cfef35c7f8f.png)

So, yeah it's open 😛

There's no more info to give, and I'm not sure how to tackle this, but the approach should be not asking us, but instead propose your PRs and someone from the team will have a look when they have time.

**Comment by mayralgr** [comment]
thanks @XhmikosR I will take a look at it and open a pr with the proposal
